weed/replication/sink/s3sink/s3_sink.go

```diff
@@ -46,40 +52,56 @@ func (s3sink *S3Sink) IsIncremental() bool {
 }
 
 func (s3sink *S3Sink) Initialize(configuration util.Configuration, prefix string) error {
-	glog.V(0).Infof("sink.s3.region: %v", configuration.GetString(prefix+"region"))
-	glog.V(0).Infof("sink.s3.bucket: %v", configuration.GetString(prefix+"bucket"))
-	glog.V(0).Infof("sink.s3.directory: %v", configuration.GetString(prefix+"directory"))
-	glog.V(0).Infof("sink.s3.endpoint: %v", configuration.GetString(prefix+"endpoint"))
-	glog.V(0).Infof("sink.s3.acl: %v", configuration.GetString(prefix+"acl"))
-	glog.V(0).Infof("sink.s3.is_incremental: %v", configuration.GetString(prefix+"is_incremental"))
+	configuration.SetDefault(prefix+"region", "us-east-2")
+	configuration.SetDefault(prefix+"directory", "/")
+	configuration.SetDefault(prefix+"keep_part_size", false)
+	configuration.SetDefault(prefix+"uploader_max_upload_parts", 1000)
```

Comment:
max is 1000 limited by s3. Better not to make this configurable.

Reply:
> max is 1000 limited by s3. Better not to make this configurable.

In fact, the default is 10,000 
```
// MaxUploadParts is the maximum allowed number of parts in a multi-part upload
// on Amazon S3.
const MaxUploadParts = 10000
```
So maybe someone will want to use it.

Reply:
Less is better.

It's confusing and error prone.

We can just fix it to 10000.

Or we can add logic to ensure the value is less than 10000.